### ISSUE TITLE
Show Nextcloud files

### ISSUE THREAD

**Issue by tailoric** [amosproj/amos2021ws03-teams-to-nextcloud#1]
## User story
1. As a user
2. I want to see my Nextcloud root directory on next cloud displayed
3. So that I can see what my files are

## Acceptance criteria
* After logging in: show the files and directories of my root directory on next cloud as a list (depends on #3)
* On click of a file or directory: Open them in a nextcloud tab

**Comment by tailoric** [comment]
Only acceptance criteria 1 is implemented
2nd one will be done in another ticket
